compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -668,18 +676,22 @@ void addMLIRAIRLoweringPasses(OpPassManager &passManager, bool packPeel) {
   {
     xilinx::air::AffineLoopOptPassOptions options;
     std::vector<unsigned> tile_sizes;
-    if (packPeel) {
+    if (passPipeline == AIEPassPipeline::PackPeelPipeline) {
       tile_sizes = {2, 2};
-    } else {
+    } else if (passPipeline == AIEPassPipeline::PadPackPipeline) {
       tile_sizes = {4, 4};
-    }
+    } else
```

Comment:
Please add a comment in the code what this code is for, and make sure that when in the future the set of passes in AIEPassPipeline changes, it is easy to understand (assert?)

Reply:
Make sense. Comments added.